.prettierrc

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I've added a config for prettier, since I would like to enable auto-formatting on save in another PR; if that is ok with you :)

The big upside of doing this is that it ensure every parts of the code keep the same clean format.

Reply:
Sounds good to me.  I'm pretty inconsistent with my code style in this project, I come from a java/c background and I tend to write all my python/javascript as if it were those languages.  If you ever want to change anything around to enforce a standard code style, feel more than free.